utils/global-state-update-gen/src/generic/config.rs

```diff
@@ -23,5 +25,27 @@ pub struct Transfer {
 pub struct AccountConfig {
     pub public_key: PublicKey,
     pub balance: Option<U512>,
-    pub stake: Option<U512>,
+    pub validator: Option<ValidatorConfig>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct ValidatorConfig {
+    pub bonded_amount: U512,
+    pub delegation_rate: u8,
+    pub delegators: Vec<DelegatorConfig>,
+}
```

Comment:
It would be nice to have a test that uses a `*.toml` file as an input and verifies this.